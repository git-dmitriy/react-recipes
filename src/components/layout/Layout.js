export const Layout = ({ children }) => {
  return (
    <main className='text-gray-600'>
      <div className='container px-5 pt-2 pb-24 mx-auto flex flex-col'>
        <div className='w-full lg:w-6/6 2xl:w-4/6 mx-auto '>{children}</div>
      </div>
    </main>
  );
};
