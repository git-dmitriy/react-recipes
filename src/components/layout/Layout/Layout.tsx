export const Layout: React.FC = ({ children }) => {
  return (
    <div className='container px-3 sm:px-5 pt-32 pb-24 mx-auto flex flex-col'>
      <div className='w-full lg:w-6/6 2xl:w-5/6 mx-auto relative'>
        {children}
      </div>
    </div>
  );
};
