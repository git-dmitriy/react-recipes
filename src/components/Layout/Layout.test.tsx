import renderer from 'react-test-renderer';
import {Layout} from '@components/Layout';
import {it, expect} from "vitest";

it('should render correctly', () => {
    const tree = renderer
        .create(
            <Layout>
                <div/>
            </Layout>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
