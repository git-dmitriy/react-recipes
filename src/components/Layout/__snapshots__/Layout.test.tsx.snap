// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render correctly 1`] = `
<div
  className="container px-3 sm:px-5 pt-32 pb-24 mx-auto flex flex-col"
>
  <div
    className="w-full lg:w-6/6 2xl:w-5/6 mx-auto relative"
  >
    <div />
  </div>
</div>
`;
